dbbuild/lib/marker.py

```diff
@@ -225,9 +225,9 @@ def __init__(self, name, chrom, rsids, index, xrefs=None, source=None):
 
     def resolve(self):
         for refr in ("GRCh37", "GRCh38"):
-            for position in self.index.resolve(self.rsids, refr, strict=True):
-                self.positions[refr].append(position)
-            self.positions[refr].sort()
+            positions = sorted(self.index.resolve(self.rsids, refr, strict=True))
+            self.positions[refr] = [pos for pos, rsid in positions]
+            self.rsids = [rsid for pos, rsid in positions]  # Possibly updated from refsnp-merged
```

Comment:
As a result, the `index.resolve` method now generates tuples instead of integers. This function is updated to sort the tuples by position so that the RSIDs and corresponding coordinates are ensured to match.